src/record/import/usecases/interface.ts

```diff
@@ -0,0 +1,8 @@
+import type { LocalRecord } from "../types/record";
+
+export type LocalRecordRepository = {
```

Comment:
@ueokande 
Hm... I have another two ideas, gateway or dataSource.
What do you think about it?

I exactly referred to the Clean Architecture, and I thought "reading files", persisting or reading data from the file system, is the responsibility of [Gateways](https://github.com/madetech/clean-architecture/blob/master/gateway.md), I think.
And I chose "Repository" because gateways are often implemented named "repository" (that is only in my search😅)
I don't know the difference between gateway and repository, but I agree the repository is often used as an abstraction of a database, and it makes us confused.

Reply:
@ueokande 
> (optional) This definition is like an interface. Why not use interface?

We, mankind, haven't made a conclusion about interface vs type...
https://zenn.dev/zaki_yama/scraps/955a2ce0b12922